### PR TITLE
Provides a guide for Lagom and Akka SBR

### PR DESCRIPTION
**Pull request by huntc**
Describes how SBR can be added to an Akka project and then illustrates how to create a network partition via ConductR infrastructure on EC2 in order to seeing SBR in action.

### CODE REVIEW

File: guides/lagom-sbr-conductr/src/main/paradox/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Deploying Lagom Microservices with Akka's Split Brain Resolver
+
+[Lagom](http://www.lagomframework.com/) is an opinionated microservices framework that makes it quick and easy to
+build, test, and deploy your systems with confidence. [Akka SBR](http://developer.lightbend.com/docs/akka-commercial-addons/current/split-brain-resolver.html) handles network partitions automatically and is a feature of the Lightbend Reactive Platform exclusively available for Lightbend customers.
```
Comment (patriknw) [inline guides/lagom-sbr-conductr/src/main/paradox/index.md]: perhaps "Akka Split Brain Resolver (SBR)"

**Review by patriknw — APPROVED** [review]
LGTM

File: guides/lagom-sbr-conductr/src/main/paradox/index.md (hunk 1 above)
Comment (huntc) [reply guides/lagom-sbr-conductr/src/main/paradox/index.md]: k

File: guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  * the entity. Each event will have an event handler registered for it, and an
+  * event handler simply applies an event to the current state. This will be done
+  * when the event is first created, and it will also be done when the entity is
+  * loaded from the database - each event will be replayed to recreate the state
+  * of the entity.
+  *
+  * This entity defines one event, the [[GreetingMessageChanged]] event,
+  * which is emitted when a [[UseGreetingMessage]] command is received.
+  */
+class HellolagomEntity extends PersistentEntity {
```
Comment (ktoso) [inline guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala:32]: `HellolagomEntity` => `HelloLagomEntity `?

Comment (huntc) [reply guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala:32]: Thanks @ktoso - I agree. I'd prefer not to change it here though as we're just using the standard Lagom getting started template.

@TimMoore FYI.

Comment (TimMoore) [reply guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala:32]: I think that's a function of how Giter8 applies your project name to the template.

Comment (TimMoore) [reply guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala:32]: Yes, it assumes that you'll separate words with spaces and not hyphens: https://github.com/foundweekends/giter8/blob/29d329bb92977a7848e8b48bd2ba5d1cf551e210/library/src/main/scala/g8.scala#L202

**Review by fsat — APPROVED** [review]

**Pull request merged**